.gitignore

```diff
@@ -0,0 +1,6 @@
+logs
+*.log
+npm-debug.log*
+package-lock.json
```

Comment:
it should be ok to have package-lock.json tracked in github: npm repo for example: https://github.com/npm/cli

Reply:
We don't need (or want) the lock file for this type of package.  This can just come out of the `.gitignore` because `.npmrc` won't let the lock file be generated anyway.  Thanks!

Reply:
yea with this being a package, we do not want a package-lock here